tests/internal/e2e/test/wdio.conf.ts

```diff
@@ -443,11 +444,28 @@ export const config: Options.Testrunner = {
         "vcastTutorial"
       );
 
-      const vcastRoot = await getVcastRoot();
-      const newVersion = "2024sp4";
+      let vcastRoot = await getVcastRoot();
+      const coded_mock_different_env_version = "2024sp1";
+      const newestRelease = "2024sp4";
```

Comment:
could we pick this up from the environment variables somehow (from those github actions env vars maybe)? Or could we define this "centrally" somewhere in the `spec_config.ts`?

Reply:
Exported from `specs_config.ts` in ce889f2f2a4161f8ff360a079bfe45911588f16e.